### ISSUE TITLE
HIP-584: Improved `StackedStateFrames` implementation by moving common structures to one-time-only creation

### ISSUE THREAD

**Issue by david-bakin-sl** [hashgraph/hedera-mirror-node#5905]
In the `com.hedera.mirror.web3.evm.store.StackedStateFrame.java` and releated StackedStateFrame classes there are instances created on every stacked state create and even every EVM frame that are in fact _fixed_ for a given run (actually, a given build), and should thus be built only once and reused.

**Comment by Nana-EC** [comment]
@david-bakin-sl what was the next item on this or was this already covered by ongoing work?

**Comment by david-bakin-sl** [comment]
TBH, I personally didn't think it was a big issue.  But I don't know the mirror node perf problems that well.  The only frame that's really affected is the `DatabaseBackedStateFrame` and that's a small instance that just holds accessors to the database, which themselves are the (slightly) more expensive things and they're only created once.  So I think there's nothing left for this issue, and I'll close it.
